js2.2.1.js

```diff
@@ -0,0 +1,38 @@
+function expression(number, operation){
+	if(!operation)
+		return number;
+	return operation(number);
+}
+
+function zero(operation) { return expression(0, operation); }
+function one(operation) { return expression(1, operation); }
+function two(operation) { return expression(2, operation); }
+function three(operation) { return expression(3, operation); }

```

Comment:
where are you giving the operation name? like expression(4,'plus') .. is this code returns the correct value?